src/egon/data/datasets/etrago_setup.py

```diff
@@ -431,9 +786,38 @@ class EgonPfHvStore(Base):
 
 
 class EgonPfHvStoreTimeseries(Base):
-    __tablename__ = "egon_etrago_store_timeseries"
-    __table_args__ = {"schema": "grid"}
+    source_dict = sources()
+    # TODO: Add other sources for dsm
+    source_list = [
+        source_dict["bgr_inspee"],
+        source_dict["bgr_inspeeds"],
+        source_dict["bgr_inspeeds_data_bundle"],
+        source_dict["bgr_inspeeds_report"],
+        source_dict["SciGRID_gas"],
```

Comment:
I think these sources are not necessary there.

Reply:
I deleted them